.github/actions/start-opensearch/action.yml

```diff
@@ -0,0 +1,128 @@
+name: 'Start OpenSearch'
+description: 'Downloads latest build of OpenSearch, installs a plugin, executes and script and then starts OpenSearch on localhost:9200'
+
+inputs:
+  platform:
+    description: 'What platform is this action running on, e.g. "windows-latest" ? '
+    required: true
+
+  opensearch-version:
+    description: 'The version of OpenSearch that should be used, e.g "3.0.0"'
+    required: true
+
+  plugin-name:
+    description: 'The the name of the plugin to use, such as security-dashboards-plugin'
+    required: true
+
+  plugin-start-script:
+    description: 'The file name for the configuration script for the plugin such as install_demo_configurations -- may not be needed for every plugin'
+    required: false
+
+  plugin-zip:
```

Comment:
I think you need to add the `plugin-zip` parameter on the workflow

#### Doesn't look like the plugin actually installed
>  yes | opensearch-plugin install file:///**docker-host/.zip**
  chmod +x plugins/opensearch-security/tools/install_demo_configuration.sh
  yes | plugins/opensearch-security/tools/install_demo_configuration.sh
  chown 1001:1001 -R /opensearch
  su -c "/opensearch/bin/opensearch" -s /bin/bash opensearch
  EOF
  docker build -t opensearch-test:latest -f- . <<EOF
  FROM ubuntu:latest
  COPY --chown=1001:1001 os-ep.sh /docker-host/
  COPY --chown=1001:1001 opensearch-security.zip **/docker-host/.zip**
  COPY --chown=1001:1001 opensearch* /opensearch/
  RUN chmod +x /docker-host/os-ep.sh

_From https://github.com/opensearch-project/security/actions/runs/3429405272/jobs/5714978189#step:5:4480_